.flake8

```diff
@@ -15,6 +15,18 @@ format = pylint
 
 exclude = .git,__pycache__
 
+per-file-ignores = \
+  # Module imported but unused
+  Base/Python/mrml.py:F401 \
+  Base/Python/slicer/logic.py:F401 \
+  Base/Python/vtkAddon.py:F401 \
+  Base/Python/vtkITK.py:F401 \
+  Base/Python/vtkSegmentationCore.py:F401 \
+  Base/Python/vtkTeem.py:F401 \
+  Modules/*/__init__.py:F401 \
+  Extensions/*/__init__.py:F401 \
+  Utilities/*/__init__.py:F401
+
 ignore = \
   # Do not use bare `except:`, it also catches unexpected events like memory errors, interrupts, system exit, and so on.
```

Comment:
We should give a rationale here. Maybe we allow catching exceptions because in an end-user application (and therefore in a high-level component) it is valid to catch all kind of errors, including memory errors, and it may be valid to handle a system exit request by interrupting the current operation and by terminating the full application.

Reply:
> We should give a rationale here. 

Agreed. This will be done while working on https://github.com/Slicer/Slicer/issues/6271 and is independent of this PR.